gossipsub.go

```diff
@@ -844,7 +844,6 @@ func (gs *GossipSubRouter) pxConnect(peers []*pb.PeerInfo) {
 		case gs.connect <- ci:
 		default:
 			log.Debugf("ignoring peer connection attempt; too many pending connections")
-			break
```

Comment:
staticcheck reports this as an ineffective break, which I don't think is true. On the other hand, why not go through the rest of the list, maybe the channel buffer will be more free as you go?

Reply:
it's useless, it can be removed.